_includes/setup/next-steps-win-linux.md

```diff
@@ -0,0 +1,5 @@
+## Next steps
+
+Now, follow our [Getting Started guide](/getting-started/)
+to run your first Flutter app on Android. To develop for iOS in addition to Android,
```

Comment:
I don't understand this next step. How many developers do we think have multiple machines with multiple different operating systems?

And even in that case, I am pretty sure they can find their way to the start of the installation instructions.

I would much rather not have the next-steps.md file be forked per OS. I think we can simply change to a single `next-steps.md` file with:

```
## Next steps

Now, follow our [Getting Started guide](/getting-started/)
to run your first Flutter app.
```